src/app/Processor/ProcessorEdit/ProcessorEdit.tsx

```diff
@@ -58,6 +56,7 @@ import { ActionModal } from "@app/components/ActionModal/ActionModal";
 import { css } from "@patternfly/react-styles";
 import { HelpIcon } from "@patternfly/react-icons";
 import { ExternalLink } from "@rhoas/app-services-ui-components";
+import StickyActionsLayout from "@app/components/StickyActionsLayout/StickyActionsLayout";
```

Comment:
Lots of changes in this file but mainly removing the layout code that now resides inside `StickyActionsLayout`, and replacing it with the new component.